app/src/main/java/com/example/gabrielsaruhashi/ramp/activities/TGuideListMapActivity.java

```diff
@@ -47,6 +43,8 @@
     Location mCurrentLocation;
     private long UPDATE_INTERVAL = 60000;  /* 60 secs */
     private long FASTEST_INTERVAL = 5000; /* 5 secs */
+    boolean isMapView = false; //default is list view
```

Comment:
could you please write a longer comment just explaining what is the purpose of this boolean (i.e keeping track of which fragment is being displayed) and also what the UPDATE_INTERVAL + FASTEST_INTERVAL more

Reply:
not totally sure what UPDATE_INTERVAL and FASTEST_INTERVAL do because I think @MasayukiNagase did that, but I'll explain the boolean more